test/gemm-microkernel-tester.h

```diff
@@ -32,6 +32,25 @@
 #include <xnnpack/post-operation.h>
 #endif  // XNN_PLATFORM_JIT
 
+inline bool IsPrime(size_t n) {
```

Comment:
These methods are unchanged - I moved them up in the header to allow use from code in this file.